src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -205,12 +217,14 @@ async function is_statement_fit_of_method_array(statements, account, method_arr,
 
 async function _is_statements_fit(statements, account, method, arn_path, req) {
     for (const statement of statements) {
+
+        dbg.log1(" statement, account, method, arn_path ", statement, account, method, arn_path);
         const action_fit = _is_action_fit(method, statement);
         const principal_fit = _is_principal_fit(account, statement);
         const resource_fit = _is_resource_fit(arn_path, statement);
         const condition_fit = await _is_condition_fit(statement, req, method);
 
-        dbg.log1('bucket_policy: is_statements_fit', action_fit, principal_fit, resource_fit, condition_fit);
+        dbg.log1('Results: action_fit, principal_fit, resource_fit, condition_fit', action_fit, principal_fit, resource_fit, condition_fit);
```

Comment:
I think the old comment is better. it specifies that its related to bucket policy

Reply:
Right, but I will keep the names of checks.